### ISSUE TITLE
/api/v1/query?query not being properly proxied to prometheus

### ISSUE THREAD

**Issue by dsolsona** [Comcast/trickster#64]
Hi,

I'm seeing a weird behaviour when sending requests to `/api/v1/query`

My setup is fairly simple, grafana -> trickster -> prometheus. I have trickster version 0.0.13 and prometheus version 1.8.2. 

Trickster is started with `trickster -log-level INFO -origin http://localhost:9090 -proxy-port 8000 -metrics-port 8001`

When I query `/api/v1/query` via trickster it returns no data

```
~$ curl 'http://localhost:8000/api/v1/query?query=aws_ec2_cpucredit_balance_minimum&time=1529991846.946&_=1529991837897'
{"status":"success","data":{"resultType":"vector","result":[]}}
```

When I send the same request to prometheus it returns the real data

```
~$ curl 'http://localhost:9090/api/v1/query?query=aws_ec2_cpucredit_balance_minimum&time=1529991846.946&_=1529991837897'
{"status":"success","data":{"resultType":"vector","result":[{"metric":{"__name__":"aws_ec2_cpucredit_balance_minimum","instance_id":"i-0e083205c45cbag46","job":"aws_ec2"},"value":[1529991846.946,"288"]}
...
...
```

Running a tcpdump on the prometheus port I can see that the query that trickster sends to Prometheus is not the same I sent to trickster
```
.#...#..GET /api/v1/query?_=1529991837897&query=aws_ec2_cpucredit_balance_minimum&time=0 HTTP/1.1
Host: localhost:9090
User-Agent: Go-http-client/1.1
Accept-Encoding: gzip
```
If I run this query against prometheus it definitely returns empty data. 

It seems that trickster is manipulating the query in the wrong way.

**Comment by frankh** [comment]
Also seeing the same issue with trickster 0.0.12 and 0.0.13

**Comment by jranson** [comment]
@dsolsona Daniel - thanks for the report. The issue is that Trickster is not passing the Time parameter through on instantaneous queries. It is something I fixed in a fork I am working on for InfluxDb support, but will go ahead and bring that fix over to master for you today.

**Comment by frankh** [comment]
Thanks @jranson!

I believe this is the PR that has fixed this in master: https://github.com/Comcast/trickster/pull/65

Is there a release cycle for trickster? When can we expect a release with this fix?

**Comment by jranson** [comment]
Hey @frankh currently I don't have a formal release cadence, I usually just spin one up when we have a significant new feature or bugfix. As this is a relatively new project, any recommendations would be great! i have not had a chance to deep dive this particular issue yet, but will definitely have a look at it in the next 24 hours and spin up a new release when the fix is confirmed.

**Comment by jranson** [comment]
Confirmed fixed in #65   thanks to @jacksontj for fixing and @dsolsona for reporting and fixing. release 0.0.14 is available on docker now and I will spin up the release here on Github in just a moment.
